src/components/card.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./card.css";
+import { SlLogin } from "react-icons/sl";
+import Stack from "react-bootstrap/Stack";
+
+function Card({ title, content, label }) {
+  const getArrowDirection = () => {
+    return content >= 150 ? "up" : "down";
+  };
+
+  const getArrowColor = () => {
+    return getArrowDirection() === "up" ? "#009E0F" : "#CC0000";
+  };
+
+  return (
+    <Stack gap={3}>
+      <div className="cardArea">
```

Comment:
We are still using custom `div` here. Can you try to convert it to use the `Card` component from react-bootstrap?

Take a look at this example - https://react-bootstrap.netlify.app/docs/components/cards#title-text-and-links

We can use this to build our own card component on top of react-bootstrap card.  Since you already have a `Card` component, you can import it as different name for example

```
import { Card as BSCard } from 'react-bootstrap';
```